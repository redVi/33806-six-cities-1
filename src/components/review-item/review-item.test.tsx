import React from "react";
import renderer from "react-test-renderer";
import ReviewItem from "@/components/review-item/review-item";

const comment = {
  id: 1,
  rating: 5,
  comment: "Nice place",
  date: "2019-06-10T02:34:08.739Z",
  user: {
    name: "Tina",
    avatarUrl: "avarat.jpg"
  }
};

describe("ReviewItem", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<ReviewItem comment={comment} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
