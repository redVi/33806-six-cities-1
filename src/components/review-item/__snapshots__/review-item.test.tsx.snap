// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewItem renders correctly 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src="avarat.jpg"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Tina
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "100%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      Nice place
    </p>
    <time
      className="reviews__time"
      dateTime="2019-06-10T02:34:08.739Z"
    >
      June 2019
    </time>
  </div>
</li>
`;
